src/Middleware/CORS/src/DisableCorsAttribute.cs

```diff
@@ -1,12 +1,19 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics;
 using Microsoft.AspNetCore.Cors.Infrastructure;
 
 namespace Microsoft.AspNetCore.Cors;
 
 /// <inheritdoc />
 [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
+[DebuggerDisplay("{ToString(),nq}")]
 public class DisableCorsAttribute : Attribute, IDisableCorsAttribute
 {
+    /// <inheritdoc/>>
+    public override string ToString()
+    {
+        return "DisableCors";
```

Comment:
I don't quite understand the advantage of returning a constant string.

Reply:
I was on the fence about doing this for attributes with no properties.

The value is for debugging and potentially being displayed in logging and places like the developer exception page, e.g. https://github.com/dotnet/aspnetcore/issues/48208

We could keep returning the type name, but someone might wonder what properties are on `DisableCorsAttribute`. Returning a string like this makes it obvious there aren't any properties.